src/am_map.c

```diff
@@ -2275,6 +2278,9 @@ void AM_drawCrosshair(int color)
 //
 void AM_Drawer (void)
 {
+  extern boolean setup_active;
```

Comment:
Could you please `include "m_menu.h"` instead? Here and in `d_main.c`?

Reply:
There's no references to either of those variables in `m_menu.h`, which means that I'd have to add these `extern` declarations to it. With this, we could also remove the `extern int menu_background` declaration in `m_misc.h`. Is that alright?

Reply:
Yes, please. We should reduce `extern` references to variables in `.c` files to a minimum. If a PR requires to add the same declaration twice, it'd better get moved into a header file instead.